plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -515,6 +518,17 @@ private String createContainer(String workspaceId,
                                            .collect(toMap(Function.identity(),
                                                           value -> new PortBinding[0])))
                   .withVolumesFrom(toArrayIfNotNull(service.getVolumesFrom()));
+        
+        if (volumesFrom != null) {
```

Comment:
It doesn't have to have access to host config. Take a look on how volumes are set [there](https://github.com/garagatyi/che/blob/5266cd930e71c3628ebbd13858676d597b3c8391/plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java#L579-L579). If it is not clear for you how to set do that, please, ping me.